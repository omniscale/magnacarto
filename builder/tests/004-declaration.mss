@num: 12;
@hash: #66cCFf;
@hash2: #66C;
@rgb: rgb(102, 0, 255);
@rgbpercent: rgb(40%, 0%, 100%);
@rgba: rgba(0, 255, 102, 0.4);
@rgbacompat: rgba(0, 255, 102, 102);
@rgbapercent: rgba(0, 100%, 40%, 40%);
@list: "Foo", "Bar", "Baz";
@listnum: 2, 3, 4;

#num            { line-width: @num; }
#hash           { line-color: @hash; line-width: 1; }
#hash2          { line-color: @hash2; line-width: 1; }
#rgb            { line-color: @rgb; line-width: 1; }
#rgbpercent     { line-color: @rgbpercent; line-width: 1; }
#rgba           { line-color: @rgba; line-width: 1; }
#rgbacompat     { line-color: @rgbacompat ; line-width: 1; }
#rgbapercent    { line-color: @rgbapercent; line-width: 1; }
#list           { text-face-name: @list; text-size: 12; text-name: "foo"; }
#listnum        { line-dasharray: @listnum; line-width: 1;}

