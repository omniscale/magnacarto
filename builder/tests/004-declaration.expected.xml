<Map srs="epsg:3857">
  <Parameters></Parameters>
  <FontSet name="fontset-1">
    <Font face-name="Foo"></Font>
    <Font face-name="Bar"></Font>
    <Font face-name="Baz"></Font>
  </FontSet>
  <Style name="num" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="hash" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#66ccff" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="hash2" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#6666cc" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="rgb" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#6600ff" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="rgbpercent" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#6600ff" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="rgba" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(0, 255, 102, 0.40000)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="rgbacompat" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(0, 255, 102, 0.40000)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="rgbapercent" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="rgba(0, 255, 102, 0.40000)" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="list" filter-mode="first">
    <Rule>
      <TextSymbolizer fontset-name="fontset-1" size="12">foo</TextSymbolizer>
    </Rule>
  </Style>
  <Style name="listnum" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke-dasharray="2, 3, 4" stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="num" srs="" status="off">
    <StyleName>num</StyleName>
  </Layer>
  <Layer name="hash" srs="" status="off">
    <StyleName>hash</StyleName>
  </Layer>
  <Layer name="hash2" srs="" status="off">
    <StyleName>hash2</StyleName>
  </Layer>
  <Layer name="rgb" srs="" status="off">
    <StyleName>rgb</StyleName>
  </Layer>
  <Layer name="rgbpercent" srs="" status="off">
    <StyleName>rgbpercent</StyleName>
  </Layer>
  <Layer name="rgba" srs="" status="off">
    <StyleName>rgba</StyleName>
  </Layer>
  <Layer name="rgbacompat" srs="" status="off">
    <StyleName>rgbacompat</StyleName>
  </Layer>
  <Layer name="rgbapercent" srs="" status="off">
    <StyleName>rgbapercent</StyleName>
  </Layer>
  <Layer name="list" srs="" status="off">
    <StyleName>list</StyleName>
  </Layer>
  <Layer name="listnum" srs="" status="off">
    <StyleName>listnum</StyleName>
  </Layer>
</Map>